src/main/java/org/folio/services/JobExecutionService.java

```diff
@@ -21,4 +25,12 @@
    */
   Future<JobExecutionCollectionDto> getCollectionDtoByQuery(String query, int offset, int limit);
 
+  /**
+   * Performs creation of JobExecution and Snapshot entities
+   *
+   * @param dto Dto contains request params enough to create JobExecution and Snapshot entities
+   * @return Future
+   */
+  Future<InitJobExecutionsRsDto> initializeJobExecutions(InitJobExecutionsRqDto dto, OkapiConnectionParams params);
```

Comment:
Maybe it's better to rename this method at all, some like initializeImport or Somethin' Else